src/classes/STG_PanelRD_CTRL.cls

```diff
@@ -81,10 +81,21 @@ public with sharing class STG_PanelRD_CTRL extends STG_Panel {
         Savepoint sp = Database.setSavepoint();
         try {
         	List<SObject> toUpdate = new List<SObject>();
-
         	if (STG_Panel.stgService.stgRD != null && !Test.isRunningTest()) {
         		toUpdate.add(STG_Panel.stgService.stgRD);
-        	}
+            }
```

Comment:
The indentation looks off here. I suspect that there may be some tabs in this class versus spaces. You should be able to convert the indentation to spaces for the entire class.